v2/datastream-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/spanner/DatastreamToSpannerExceptionClassifier.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates.spanner;
+
+import com.google.cloud.teleport.v2.spanner.migrations.exceptions.SpannerMigrationException;
+import com.google.cloud.teleport.v2.spanner.migrations.exceptions.SpannerMigrationException.ErrorCode;
+import java.util.Set;
+
+/* Helper class to classify SpannerExceptions to Retryable error or Permanent error */
```

Comment:
add on in the docs to clarify why this is specific to a template ?

Reply:
Done.